.vscode/settings.json

```diff
@@ -1,10 +1,17 @@
 {
-    "python.formatting.provider": "black",
     "python.testing.pytestArgs": [
         "tests"
     ],
+    "python.formatting.provider": "black",
     "python.testing.unittestEnabled": false,
     "python.testing.pytestEnabled": true,
     "python.linting.pylintEnabled": true,
-    "python.linting.flake8Enabled": true
+    "python.linting.flake8Enabled": true,
+    "[python]": {
+        "editor.formatOnSave": true,
+        "editor.codeActionsOnSave": {
+            "source.organizeImports": true
+        },
+        "editor.defaultFormatter": "ms-python.python",
+    },
```

Comment:
Please remove the settings related to autoformating. We are using Black, and this can be configured to reformat the code when it is run.

Reply:
I have just modified the file so I keep the original config! 

I think we already discussed it in the previous meeting, but I am not able to format files in VSCode with black with the original config (not even using the command _make _form__ or any other alternative). 

Maybe I am just wrong, but if you do not mind, we can talk about it tomorrow.